app/views/building_map/index.html.erb

```diff
@@ -9,3 +9,46 @@
     polylines: BuildingMapHelper.leaflet_polylines(@route),
     markers: BuildingMapHelper.leaflet_markers(@route, @target))
 %>
+<style>
+.indoorLabel{
+  background: rgba(255, 255, 255, 0);
+  border:0;
+  border-radius:0px;
+  box-shadow: 0 0px 0px;
+}
+</style>
+<script>
+  function onEachFeature(feature, layer) {
```

Comment:
Rename this function to `addIndoorLabel` or sth. like this.
Alternatively this could also be a lambda function.